src/mscorlib/src/System/OperatingSystem.cs

```diff
@@ -138,6 +138,9 @@ private OperatingSystem()
                     case PlatformID.MacOSX:
                         os = "Mac OS X ";
```

Comment:
That would be off the topic for this commit and seems worthy to become another issue and PR. Note that there are:

```
        Win32S        = 0,
        Win32Windows  = 1,
        Win32NT       = 2,
        WinCE         = 3,
        Unix          = 4,
        Xbox          = 5,
        MacOSX        = 6
```